pkgs/desktops/gnome-2/platform/gnome-vfs/default.nix

```diff
@@ -20,6 +20,8 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = [ GConf glib ];
 
+  patches = [ ./gnome-vfs-2.24.4-openssl-1.1.patch ];
```

Comment:
Could you use `fetchpatch` on https://bugs.gentoo.org/attachment.cgi?id=535944, instead of adding the patch in tree, and also add a comment explaining the issue and linking to https://bugs.gentoo.org/592540?

Reply:
done!